tests/wallet/test_singleton.py

```diff
@@ -50,12 +51,10 @@ def test_only_odd_coins():
             [],
         ]
     )
-    try:
+
+    with pytest.raises(Exception) as exception_info:
```

Comment:
instead of capturing the `exception_info` and checking it, you can add `match="clvm raise" to the call to `raises()`

Reply:
I find this pattern with the values separate to be more explicit and less error prone than having the exception encode the 'name' and number to a string and using a regex to match that.